File: Structural/Bridge/Python/basic_remote.py
# basic_remote.py

from remote import Remote

class BasicRemote(Remote):
  def volume_up(self):
    self.device.set_volume(self.device.get_volume() + 10)
  def volume_down(self):
    self.device.set_volume(self.device.get_volume() - 10)

File: Structural/Bridge/JavaScript/basicRemote.js
// basicRemote.js

const Remote = require("./remote");

class BasicRemote extends Remote {
  volumeUp() {
    this.device.setVolume(this.device.getVolume() + 10);
  }

  volumeDown() {
    this.device.setVolume(this.device.getVolume() - 10);
  }
}

module.exports = BasicRemote;
